requirements.txt

```diff
@@ -20,7 +20,7 @@ pdfkit
 phenopackets
 xlsxwriter
 click
-cryptography<3.4
+cryptography
```

Comment:
Ahem, that was intended for a separate thaw experiment. Seems good to go, but since we may or may not keep this branch I'll move it.